open-company/open-company-web
# Onboard fields fix

Fix: the onboard form sometimes is updated after the user has started typing. That happens because the user data are loaded with some delay and once that happens the values from the server are inserted in the fields and the name one is focused.

To test:
- onboard via Slack
- once you land on team setup lander (after granting Slack permissions) start typing ASAP in the Why carrot field
- did it NOT move the focus to the name field after you started typing?
- check that inviting via team email link still works and is not affected by the above issue (start typing first name asap)
- check that inviting via email still works and is not affected by the above issue (start typing first name asap and check your input is not overwritten)
